METRICS.md

```diff
@@ -0,0 +1,574 @@
+<!--
+http://www.apache.org/licenses/LICENSE-2.0.txt
+
+
+Copyright 2017 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Snap Ceph Perf Counters Collector Plugin
+## Full metric list for Ceph 11.0.2
+Metric list was generated dynamically by plugin and retrieved using command:
+```
+snaptel metric list --verbose
+```
+
+NAMESPACE | VERSION | UNIT | DESCRIPTION
```

Comment:
1) Based on the amount of metrics, consider keeping the form as it was done before - I mean splitting those metrics per OSD, MON and MDS. 
2) Consider trimming the prefix of namespaces (as it was done here [MDS_PERFCNT.md#25](https://github.com/intelsdi-x/snap-plugin-collector-ceph/pull/25/files#diff-3745e6fb21297bd2c19a57c09a0638f8L25)), so they will be shorter and more readable. 
3) In my opinion, the column with the version of plugin should be removed.
4) Please, add notice about that metrics depend i.a. on Ceph version and might vary. Keep mention about the version for which this listed metrics were retrieved (as you have at the moment)